test/base/ProcessJson.scala

```diff
@@ -0,0 +1,1941 @@
+      |}
+    """.stripMargin
+  )
+
+  val validQuestionStanzaJson: JsValue = Json.parse(
+    """
+      |{
+      |    "type": "QuestionStanza",
+      |    "text": 3,
+      |    "answers": [
+      |        4,
+      |        5
+      |      ],
+      |     "next": [
+      |        "5",
+      |        "19"
+      |      ],
+      |     "stack": false
+      |}
+    """.stripMargin
+  )
+
+  val validGenericStanzaJson: JsValue = Json.parse(
+    """
+      |{
+      |    "type": "EndStanza"
+      |}
+    """.stripMargin
+  )
+
+  val validOnePageJson: JsValue = Json.parse(
+    """
+      |{
+      |  "meta": {
+      |    "title": "Customer wants to make a cup of tea",
+      |    "id": "oct90001",
+      |    "ocelot": 1,
+      |    "lastAuthor": "000000",
+      |    "lastUpdate": 1500298931016,
+      |    "version": 4,
+      |    "filename": "oct90001.js"
+      |  },
+      |  "howto": [],
+      |  "contacts": [],
+      |  "links": [],
+      |  "flow": {
+      |    "1": {
+      |      "type": "ValueStanza",
+      |      "values": [
+      |        {
+      |          "type": "scalar",
+      |          "label": "pageUrl",
+      |          "value": "/feeling-bad"
+      |        }
+      |      ],
+      |      "next": ["2"],
+      |      "stack": true
+      |    },
+      |    "2": {
+      |      "type": "InstructionStanza",
+      |      "text": 0,
+      |      "next": [
+      |        "3"
+      |      ],
+      |      "stack": true
+      |    },
+      |    "3": {
+      |      "type": "InstructionStanza",
+      |      "text": 1,
+      |      "next": [
+      |        "4"
+      |      ],
+      |      "stack": true
+      |    },
+      |    "3": {
+      |      "type": "QuestionStanza",
+      |      "text": 3,
+      |      "answers": [
+      |        4,
+      |        5
+      |      ],
+      |      "next": [
+      |        "end",
+      |        "end"
+      |      ],
+      |      "stack": false
+      |    },
+      |    "end": {
+      |      "type": "EndStanza"
+      |    }
+      |  },
+      |  "phrases": [
+      |    ["Ask the customer if they have a tea bag", "Welsh, Ask the customer if they have a tea bag"],
+      |    ["Do you have a tea bag?", "Welsh, Do you have a tea bag?"],
+      |    ["Yes - they do have a tea bag", "Welsh, Yes - they do have a tea bag"],
+      |    ["No - they do not have a tea bag", "Welsh, No - they do not have a tea bag"],
+      |    ["Ask the customer if they have a cup", "Welsh, Ask the customer if they have a cup"],
+      |    ["Do you have a cup?", "Welsh, Do you have a cup?"],
+      |    ["yes - they do have a cup ", "Welsh, yes - they do have a cup "],
+      |    ["no - they don’t have a cup", "Welsh, no - they don’t have a cup"]
+      |  ]
+      |}
+    """.stripMargin
+  )
+
+  val invalidJson: JsValue = Json.parse(
+    """
+      |{
+      | "invalidJson": "invalid Json"
+      |}
+    """.stripMargin
+  )
+
+  val fullJson: JsValue = Json.parse(
```

Comment:
The JSON is fine for testing actual processes but to test parsing logic is unessasaily large. Possibly have multiple valid JSON payloads named after the purpose would be easier for new people to understand/maintain?